apps/condo/domains/common/hooks/useCurrentBuild.tsx

```diff
@@ -0,0 +1,21 @@
+import useSWR from 'swr'
+import get from 'lodash/get'
+import getConfig from 'next/config'
+
+const {
+    publicRuntimeConfig: { serverUrl },
+} = getConfig()
+
+const REQUEST_KEY = '/api/version'
```

Comment:
```suggestion
const REQUEST_PATH = '/api/version'
```

Reply:
According to the [documentation](https://www.npmjs.com/package/swr), this is actually a `key`

Reply:
Yes, it's a key to store cached data.